### DISCUSSION TITLE
User is getting error as "Form upload failed" when upload the odk form

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [samagra-comms/community#67] [General Discussion]
@abhay-samagra @ ChakshuGautam @ aayushkedia95 @SaketSinha 

**Steps:**

- Open sunbird ED staging url
- Login with orgadmin credential
- Click on profile icon
- Click on communication console
- Click on add new button
- Fill the all mandatory fields then click on next button
- Click on add logic button
- Fill the name and upload the odk form 
- Observe

**Actual result :** User is getting error as  "Form upload failed" when upload the odk form
**Expected result :** User should able to upload odk form successfully

Please refer jira link for more details - https://project-sunbird.atlassian.net/browse/ED-2808

**Comment by ashokreddy1208** [comment]
@abhay-samagra Waiting for your response, with is a blocker for us to signoff 6.0 sunbird ED release.

cc @chitranshu-keshav @vinukumar-vs @rajeevsathish

**Comment by abhay-samagra** [comment]
Yes, Ashok we have noted this issue.

Will coordinate internally and let you know the ETA for resolution.

Thanks and Regards

Abhay Singh

Senior Product Consultant | +91 787 477 4342

Samagra | Transforming Governance

Address: 9/5 Sarvapriya Vihar, New Delhi-16
***@***.***,77.1994501,17z/data=!3m1!4b1!4m5!3m4!1s0x390ce273981e5dfb:0x7bfceb198af17562!8m2!3d28.5566128!4d77.2016388?shorturl=1>

Website: www.samagragovernance.in

Follow Us: LinkedIn
<https://www.linkedin.com/company/samagra-transforming-governance/> |
YouTube <https://www.youtube.com/channel/UCfkXErS-f87xUQkmSKSC8bg> |
Instagram <https://www.instagram.com/samagragovernance/?hl=en>

**Comment by karntrehan** [comment]
@chinmoy12c @RyanWalker277 Lets debug this.

**Comment by ashokreddy1208** [comment]
@karntrehan @abhay-samagra please let us know the steps to fix this issue.

**Reply by chinmoy12c** [reply]
As explained here: https://project-sunbird.atlassian.net/browse/ED-2808?focusedCommentId=113245

There is no issue with the backend system, the form just needs to be updated with a different form id and version.

**Comment by abhay-samagra** [comment]
Hi Chinmoy,Please provide an update to the Sunbird team.RegardsAbhayOn 08-Sep-2023, at 1:24 PM, ashokreddy1208 ***@***.***> wrote:﻿
@karntrehan @abhay-samagra please let us know the steps to fix this issue.
